### ISSUE TITLE
Should not stop if file do not exist

### ISSUE THREAD

**Issue by PyrosBrother** [munyabe/FormatAllFiles#9]
Hello,
This extension is great but if a file included in project is missing. It stop, then you have to remove the file and run again format all files and it could be boring on huge project.
It would be nice if it did not stop and put a error on console log and continue with next file

Regards

**Comment by HumanGamer** [comment]
I agree, I'm working on a large project and this is a problem. I might try to implement it myself, however, I've never worked on a VS Extension before.
